# generated by patch-package 6.4.8 on 2022-03-30 17:15:33
#
# command:
#   npx patch-package mdx-bundler
#
# declared package:
#   mdx-bundler: 8.0.1
#
diff --git a/node_modules/mdx-bundler/dist/client.js b/node_modules/mdx-bundler/dist/client.js
index d1d60db..fb41aea 100644
--- a/node_modules/mdx-bundler/dist/client.js
+++ b/node_modules/mdx-bundler/dist/client.js
@@ -8,7 +8,7 @@ exports.getMDXExport = getMDXExport;
 
 var React = _interopRequireWildcard(require("react"));
 
-var _jsx_runtime = _interopRequireWildcard(require("react/jsx-runtime.js"));
+var _jsx_runtime = _interopRequireWildcard(require("react/jsx-runtime"));
 
 var ReactDOM = _interopRequireWildcard(require("react-dom"));
 
